python/lsst/ctrl/mpexec/singleQuantumExecutor.py

```diff
@@ -144,8 +180,10 @@ def checkExistingOutputs(self, quantum, butler, taskDef):
         Returns
         -------
         exist : `bool`
-            True if all quantum's outputs exist in a collection and
-            ``skipExisting`` is True, False otherwise.
+            True if a previous execution of this quanta appears to have
+            completed successfully (either because metadata was written or all
+            datasets were written) and ``skipExisting`` is `True`, `False`
```

Comment:
im not sure the bit about skipExisting is clear in this doc string

Reply:
Will reword.